_sass/layout/_navigation.scss

```diff
@@ -18,19 +19,13 @@ body.nav-is-open {
 }
 
 .sticky-wrapper.is-sticky > .header {
+  transition: all .5s ease-in-out;
   background: $nav-sticky-bg-color;
-  -webkit-transition: all 0.5s ease-in-out;
```

Comment:
@bulletfactory Github pages has a Jekyll plugin for auto-prefixing CSS, so all you need to do is add the un-prefixed property and Jekyll will handle the rest!